exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -120,7 +120,7 @@ drill.exec: {
     affinity.factor: 1.2
   },
   sys.store.provider: {
-    class: "org.apache.drill.exec.store.sys.zk.ZkPStoreProvider",
+    class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
```

Comment:
I second on Sudheesh's concern. Unless there is a strong reason that we have to make a breaking change such that "all references to ZkPStoreProvider will fail following this patch", I feel we need to think twice before we make such change.  Is this the only way that we can go? Have we assessed the impact to Drill's current users (After all, Drill is post 1.0 release)? Do we have a well-documented plan to help them to migrate, if the backward compatibility is broken in new release?